npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -72,6 +72,29 @@ function modifyWebpackConfigForCypress (webpackConfig: Partial<Configuration>) {
   return webpackConfig
 }
 
+async function addEntryPoint (webpackConfig: Configuration) {
```

Comment:
Angular includes global styles and polyfills via the `webpackConfig.entry` field. We were overriding the entry with our Cypress entrypoint, so I refactored the handling of entrypoints to preserve additional entrypoints and deleted unneeded entrypoints from the other framework handlers.